query-connector/README.md

```diff
@@ -4,84 +4,53 @@
 
 The DIBBs Query Connector app offers a REST API for searching for a patient and viewing information tied to your case investigation.
 
-### Running Query Connector
+### Running Query Connector (Prerequisites)
```

Comment:
Think since the dev commands run the db in docker, docker installed is a prereq for any way of running QC